src/types/onyx/RecentlyUsedCategories.ts

```diff
@@ -0,0 +1,3 @@
+type RecentlyUsedCategories = Array<typeof String>;
```

Comment:
Hmm I _think_ it would be
```suggestion
type RecentlyUsedCategories = string[];
```
but I'll let @hayata-suenaga weigh in 😅